store/index.js

```diff
@@ -146,6 +149,54 @@ export const getters = {
         return p_state.categories[p_category].componentName;
     },
 
+    getCategoricalJsonOutput: (p_state) => (p_columnName) => {
+
+        // 0. Initialize output object
+        const annotatedDictColumn = p_state.dataDictionary.annotated[p_columnName];
+        const category = p_state.columnToCategoryMap[p_columnName];
+        const formattedOutput = {
+
+            Annotations: {
+
+                IsAbout: {
+
+                    Label: "",
+                    TermURL: ""
+                },
+                Levels: {},
+                MissingValues: []
+            }
+        };
+
+        // 1. Fill out Annotations 'IsAbout' section
+
+        // A. Label matches the assigned category
+        formattedOutput.Annotations.IsAbout.Label = category;
+
+        // B. Term matches the category identifier
+        formattedOutput.Annotations.IsAbout.TermURL = p_state.categories[category].identifier;
+
+        // 2. Fill out Annotations 'Levels' section
+        Object.keys(annotatedDictColumn.valueMap).forEach(rawValue => {
+
+            formattedOutput.Annotations.Levels[rawValue] = {};
+
+            p_state.categoricalOptions[category].forEach(option => {
```

Comment:
couldn't we just do 

```js
const myOption = p_state.categoricalOptions[category].find(option => option.identifier === annotatedDictColumn.valueMap[rawValue])
```
here? And then do the assignment in one go with the object declaration above, i.e.:

```js
formattedOutput.Annotations.Levels[rawValue] = {
"Label": myObject.label
"TermURL": myObject.identifier
}
```

Reply:
I think so